src/sundials/sundials_errors.c

```diff
@@ -0,0 +1,91 @@
+/* -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2022, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------*/
+
+#include <stdlib.h>
+#include <string.h>
+#include <sundials/sundials_core.h>
+#include <sundials/impl/sundials_errors_impl.h>
+
+static inline char* combineFileAndLine(int line, const char* file)
+{
+  size_t total_str_len = strlen(file) + 6; /* TODO(CJB): need to figure out width of line */
```

Comment:
I would think of this as a "lowest possible" priority, since there's no issue with having a few extra spaces, and we have no files with over 999,999 lines of code.
```suggestion
  size_t total_str_len = strlen(file) + 6;
```